examples/worlds/multi_lrauv_race.sdf

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" ?>
 
 <!-- Run demo with ign-gazebo examples/standalone/multi_lrauv_race
-  (example is provided in ign-gazebo6+). -->
+  (example is provided in ign-gazebo7+). -->
```

Comment:
I think this line can be removed, because the examples folder should match the world file for a given branch

Reply:
reverted in https://github.com/ignitionrobotics/ign-gazebo/pull/1183/commits/accf027d3417674dd2f48fec374596e723015443